src/Components/CreateDog/CreateDog.js

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import "./CreateDog.styles.scss";
+import axios from "axios";
+
+function DogCardInfo() {
+  const [dogName, setDogName] = useState([]);
```

Comment:
you are uring an array so change to plurar

Reply:
Changed both variables to plural for better understanding